packages/pluggableWidgets/image-viewer-web/CHANGELOG.md

```diff
@@ -4,8 +4,9 @@ All notable changes to this widget will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
 ### Changed
 - Converted into a pluggable widget.
 
 ## Older releases
-See [marketplace](https://marketplace.mendix.com/link/component/108261) for previous releases.
\ No newline at end of file
+See [marketplace](https://marketplace.mendix.com/link/component/108261) for previous releases.
```

Comment:
What changed here 🤔

Reply:
New line in the end, but github doesnt show..

Reply:
🤯